stickermanager.py

```diff
@@ -0,0 +1,276 @@
+import os
+import json
+import asyncio
+from time import time
+from io import BytesIO
+from html import escape as escape_html
+from typing import Union, NamedTuple, Callable, Optional, Tuple
+
+from PIL import Image
+
+from telethon.tl.custom import Message, Button
+from telethon.tl.types import (
+    InputStickerSetID, InputStickerSetItem, InputDocument,
+    InputMediaUploadedDocument, InputPeerSelf, DocumentAttributeAnimated
+)
+from telethon.tl.types.messages import StickerSet
+from telethon.tl.functions.stickers import CreateStickerSetRequest, AddStickerToSetRequest
+from telethon.tl.functions.messages import UploadMediaRequest
+from telethon import events, utils, TelegramClient
+
+POLL_TEMPLATE = (
+    '<a href="tg://user?id={sender_id}">{sender_name}</a> has suggested this '
+    'sticker be added to <a href="{pack_link}">the group\'s sticker pack</a> '
+    'with the emoji {emoji}.\n\n'
+    '<strong>Current result: {score}</strong>\n'
+    'For: {yes}\n'
+    'Against: {no}'
+)
+
+POLL_FINISHED_TEMPLATE = (
+    '<strong>This sticker poll finished with a final score of {score}. {result}</strong>\n\n'
+    '<a href="tg://user?id={sender_id}">{sender_name}</a> had suggested this '
+    'sticker be added to the group\'s sticker pack with the emoji {emoji}.\n\n'
+    'For: {yes}\n'
+    'Against: {no}'
+)
+
+VOTE_TEMPLATE = '<a href="tg://user?id={uid}">{displayname}</a> ({weight})'
+
+RESULT_ADDED = 'The sticker has been added to <a href="{pack_link}">the pack</a>.'
+RESULT_REJECTED = 'The sticker was rejected from <a href="{pack_link}">the pack</a>.'
+
+UP = '\U0001f53c'
+DOWN = '\U0001f53d'
+
+VoteData = NamedTuple('VoteData', weight=int, displayname=str)
+Scores = NamedTuple('Scores', sum=int, yes=int, no=int)
+
+current_vote: Optional[dict] = None
+current_vote_lock: asyncio.Lock = asyncio.Lock()
+sticker_pack = None
+
+with open(os.path.join(os.path.dirname(__file__), 'stickermanager.tsv')) as f:
+    WEIGHTS = {
+        int(uid) if uid.strip().isnumeric() else uid:
+            (0 if weight == '-' else float(weight), None if name == '-' else name)
+        for uid, weight, name
+        in (line.strip().split('\t', 2)
+            for line in f
+            if len(line) > 0 and not line[0] == '#' and not line.isspace())
```

Comment:
Python prefers using truthy values (`if line`) when possible.

Reply:
Normally `line.startswith` makes the intention clearer, but it's okay to leave the check against the first character.